src/plugins/intel_npu/src/plugin/npuw/partitioning/online/snapshot.cpp

```diff
@@ -342,10 +424,18 @@ void Snapshot::repeatedBlocks() {
 
     identifyUniques();
     repeat([&] {
-        mergeUniques();
+        repeat([&] {
+            mergeUniques();
+        });
+        mergeTriangles();  // FIXME: assuming that w/o a particular set of properties (isolate, nofold) this pass does
+                           // nothing
```

Comment:
Should this actually be fixed?

Also, if the comment is here to stay, i'd just put it on the previous line - the current construct looks weird.

Reply:
Removed